src/main/java/net/javadiscord/javabot/systems/tags/commands/CustomTagsAdminCommand.java

```diff
@@ -3,17 +3,18 @@
 import com.dynxsty.dih4jda.interactions.commands.SlashCommand;
 import net.dv8tion.jda.api.interactions.commands.DefaultMemberPermissions;
 import net.dv8tion.jda.api.interactions.commands.build.Commands;
+import net.javadiscord.javabot.systems.moderation.CommandModerationPermissions;
 
 /**
  * Represents the `/tag-admin` command. This holds administrative commands for managing "Custom Tags".
  */
-public class CustomTagsAdminCommand extends SlashCommand {
+public class CustomTagsAdminCommand extends SlashCommand implements CommandModerationPermissions {
 	/**
 	 * This classes constructor which sets the {@link net.dv8tion.jda.api.interactions.commands.build.SlashCommandData} and
 	 * adds the corresponding {@link net.dv8tion.jda.api.interactions.commands.Command.Subcommand}s.
 	 */
 	public CustomTagsAdminCommand() {
-		setSlashCommandData(Commands.slash("tag-admin", "Administrative commands for managing \"Custom Tags\".")
+		setModerationSlashCommandData(Commands.slash("tag-admin", "Administrative commands for managing \"Custom Tags\".")
```

Comment:
Note that this overrides `.setDefaultPermissions(DefaultMemberPermissions.DISABLED)` and makes the command accessible to non-admin staff.